app/views/idv/review/new.html.slim

```diff
@@ -32,7 +32,9 @@ p.italic = t('idv.messages.review.intro')
   .h6 Phone number
   .h4.bold #{idv_params[:phone]}
 
-= form_tag('/idv/review', method: 'put')
+= simple_form_for(current_user, url: idv_review_path,
+    html: { autocomplete: 'off', method: :put, role: 'form' }) do |f|
+  = f.input :password, label: t('idv.form.password'), required: 'true'
```

Comment:
if `password` is a required field for the ActiveRecord obj, Simpleform will mark as required automagically. If not, I think we can use `true` instead of `'true'` here